clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -65,6 +65,24 @@ func (o *ObjectTransitionTimes) Get(key, phase string) (time.Time, bool) {
 	return time.Time{}, false
 }
 
+// DeleteKey removes all entries for given key.
+func (o *ObjectTransitionTimes) DeleteKey(key string) {
+	if _, exists := o.times[key]; exists {
+		delete(o.times, key)
+	}
+}
+
+// Count returns number of key having given phase entry.
+func (o *ObjectTransitionTimes) Count(phase string) int {
```

Comment:
I was wondering if we can avoid that by making things differently, but didn't come up with anything reasonable enough :(

Reply:
Probably for simplicity you can move it to a separate PR :)